framework/src/CMakeLists.txt

```diff
@@ -63,33 +63,33 @@ set(_srcs
 
 set(_private_headers
   util/FrameworkPrivate.h
-  util/LDAPExpr_p.h
-  util/Properties_p.h
-  util/Utils_p.h
+  util/LDAPExpr.h
+  util/Properties.h
+  util/Utils.h
 
-  service/ServiceHooks_p.h
-  service/ServiceListenerEntry_p.h
-  service/ServiceListenerHook_p.h
-  service/ServiceListeners_p.h
+  service/ServiceHooks.h
+  service/ServiceListenerEntry.h
+  service/ServiceListenerHook.h
+  service/ServiceListeners.h
   service/ServiceReferenceBasePrivate.h
   service/ServiceRegistrationBasePrivate.h
-  service/ServiceRegistry_p.h
+  service/ServiceRegistry.h
 
-  bundle/BundleArchive_p.h
+  bundle/BundleArchive.h
   bundle/BundleContextPrivate.h
-  bundle/BundleHooks_p.h
-  bundle/BundleManifest_p.h
+  bundle/BundleHooks.h
+  bundle/BundleManifest.h
   bundle/BundlePrivate.h
-  bundle/BundleRegistry_p.h
-  bundle/BundleResourceContainer_p.h
-  bundle/BundleStorage_p.h
-  bundle/BundleStorageFile_p.h
-  bundle/BundleStorageMemory_p.h
-  bundle/BundleThread_p.h
-  bundle/BundleUtils_p.h
-  bundle/CoreBundleContext_p.h
-  bundle/Debug_p.h
-  bundle/Fragment_p.h
-  bundle/ModuleObjFile_p.h
-  bundle/Resolver_p.h
+  bundle/BundleRegistry.h
+  bundle/BundleResourceContainer.h
+  bundle/BundleStorage.h
+  bundle/BundleStorageFile.h
+  bundle/BundleStorageMemory.h
+  bundle/BundleThread.h
+  bundle/BundleUtils.h
+  bundle/CoreBundleContext.h
+  bundle/Debug.h
+  bundle/Fragment.h
+  bundle/ModuleObjFile.h
+  bundle/Resolver.h
```

Comment:
Thanks for catching this, Sascha. Also, I didn't realize that you wanted to remove the "_p" suffix from all headers. Awesome!